extensions/jni/jvm/JniReferenceObjects.h

```diff
@@ -137,12 +137,12 @@ class JniByteInputStream : public minifi::InputStreamCallback {
     }
 
     // seek to offset
-    int remaining = size;
+    gsl_Expects(size >= 0);
```

Comment:
I would prefer preconditions to be at the start of the function whenever possible.

Reply:
Fixed in 9adc6ba9612a2d1ad3543d7812709b0ae9b1f8e1